.github/workflows/test_package.yml

```diff
@@ -26,7 +28,7 @@ jobs:
     - name: Set min. dependencies
       if: matrix.requires == 'minimal'
       run: |
-        python -c "req = open('requirements/requirements.txt').read().replace('>', '=') ; open('requirements/requirements.txt', 'w').write(req)"
+        python -c "req = open('setup.cfg').read().replace(' >= ', ' == ') ; open('setup.cfg', 'w').write(req)"
```

Comment:
This is not very robust: it is based on the assumption that other table content values that are not dependencies (e.g. the minimum Python version required) do not have whitespaces around them. Finding the way to improve it (it may require an additional script to avoid cluttering the workflow YAML file) is left for a separate PR.